content/en-us/docs/developer/run-a-local-development-network.md

```diff
@@ -32,91 +32,209 @@ It's required to have at least 4 cores and 8GB RAM to build the project includin
-  <div class="tab-pane fade" id="nav-contact" role="tabpanel" aria-labelledby="nav-contact-tab">...</div>
-</div>
+  <div class="tab-pane" id="profile" role="tabpanel" aria-labelledby="profile-tab">
+  <br>
 
-### Linux
+Install the system level dependencies for Fedora
 
-#### Install Dependencies (Linux)
+```bash
+sudo dnf update
+sudo dnf install make automake gcc gcc-c++ kernel-devel ocaml autoconf automake libtool wget python openssl-devel git cmake perl pkg-config curl
+```
 
-- Run the script to install and check required dependencies for Phala
+Ensure `clang` exists in your $PATH by executing
 
-Download the script (successfully tested on Ubuntu 20.04):
 ```bash
-wget https://raw.githubusercontent.com/hauni97/phala_quick_run/main/phala_quick_install.sh
+which clang
+# /usr/bin/clang
+
+# by default, the clang executable is installed as "clang-10"
+# if you got `clang not found`, you need to link the clang-10 to clang
+sudo ln -sf /usr/bin/clang-10 /usr/bin/clang
 ```
-Make it executable:
+
+> Notes on LLVM: We require at least LLVM-9, but higher versions are also supported. An older version like LLVM 6.0 breaks the core blockchain compilation.
+
+Install Rust. Please choose the default toolchain
+
 ```bash
-chmod +x phala_quick_install.sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source $HOME/.cargo/env
 ```
-Excute it to automate the next three manual steps to get the environment ready:
+
+Install Intel SGX SDK
+
 ```bash
-sudo ./phala_quick_install.sh
+wget https://download.01.org/intel-sgx/sgx-linux/2.14/distro/fedora31-server/sgx_linux_x64_sdk_2.14.100.2.bin
+chmod +x sgx_linux_x64_sdk_2.14.100.2.bin
+echo -e 'no\n/opt/intel' | sudo ./sgx_linux_x64_sdk_2.14.100.2.bin
+source /opt/intel/sgxsdk/environment
 ```
 
-![](/images/docs/auto-install.gif)
+> You can add `source /opt/sgxsdk/environment` to your `~/.bashrc` (or `~/.zshrc`, depends on which shell you're using).
 
-(Walkthough of how a successfull environement setup looks like when script is run)
+Install Node.js (>= v12) & yarn
 
-#### Manual Installation (Linux)
+```bash
+curl -sL https://rpm.nodesource.com/setup_16.x | sudo -E bash -
+sudo dnf install -y nodejs
+sudo npm install -g yarn
+```
 
-- Install the system level dependencies (manual steps)
+  </div>
+  <div class="tab-pane" id="messages" role="tabpanel" aria-labelledby="messages-tab">
+  <br>
 
-  ```bash
-  sudo apt update
-  sudo apt install -y build-essential ocaml ocamlbuild automake autoconf libtool wget python libssl-dev git cmake perl pkg-config curl llvm-10 clang-10 libclang-10-dev
-  ```
+> Instructions for (but not limited to) a relatively clean Windows11/WSL2 environment (Ubuntu 20.04LTS, and git correctly installed) are preceded by the mention "For WSL2"
+>> Install WSL2 (Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11)
+>>More infos at: https://docs.microsoft.com/en-us/windows/wsl/install
```

Comment:
I suggest linking it here instead of the using the plain URL.

Reply:
Updated to reflect a link on `Install WSL2`
![image](https://user-images.githubusercontent.com/64296537/144766528-33e27f3c-0b57-46dc-800c-31cc0036ddf5.png)